build.gradle

```diff
@@ -1,27 +1,33 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.4.30'
+    ext {
+        kotlin_version = '1.4.30'
```

Comment:
Nitpick here, kotlin can be bumped to 1.5 but I assume you were probably going to do that anyway.

Reply:
I'll do that later, `1.5` wasn't released when I've started with this PR